pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -526,11 +543,14 @@ type connectionInfo struct {
 func (ku *KubeUtil) init() error {
 	var err error
 
+	// Kubelet is unavailable, proxying calls through the APIServer - EKS on Fargate.
+	if config.Datadog.GetBool("eks_fargate") {
+		ku.kubeletProxiedEndpoint = true
+	}
```

Comment:
why not `ku.kubeletProxiedEndpoint = config.Datadog.GetBool("eks_fargate")` ?

Reply:
I wanted to increase my github stats for # of lines in my PRs (jk, good point)